cmd/noms/noms_log.go

```diff
@@ -61,18 +64,27 @@ func setupLogFlags() *flag.FlagSet {
 func runLog(args []string) int {
 	useColor = shouldUseColor()
 	cfg := config.NewResolver()
-	database, value, err := cfg.GetPath(args[0])
 
-	if err != nil {
-		d.CheckErrorNoUsage(err)
+	resolved := cfg.ResolvePathSpec(args[0])
+	sp, err := spec.ForPath(resolved)
+	d.CheckErrorNoUsage(err)
+	pinned, ok := sp.Pin()
+	if !ok {
+		fmt.Fprintf(os.Stderr, "Cannot resolve spec: %s\n", args[0])
+		return 1
 	}
+	database := pinned.GetDatabase()
```

Comment:
`defer pinned.Close()` (not `defer database.Close()`)
In fact we should also have a `defer sp.Close()` too.